packages/calling-stateful-client/review/beta/calling-stateful-client.api.md

```diff
@@ -307,6 +314,13 @@ export interface TranscriptionCallFeature {
     isTranscriptionActive: boolean;
 }
 
+// @beta
+export interface TransferFeature {
+    acceptedTransfers: {
```

Comment:
Only when we accept a new transfer. That is a good question though. Maybe changing the TransferFeature object to the following would be better.
```
export interface TransferFeatureState {
  transferRequests: TransferRequest[]
}

export interface TransferRequest {
  timestamp: Date
  acceptedCallId?: string
}
```

Reply:
Re: timestamp, is this the timestamp the request came in, or the timestamp that the request was accepted?

Reply:
We should have both in this case

Reply:
Actually, this does not work because I don't have anything to index the TransferRequest object with such that when the TransferRequest I am able to index the TransferRequest to set the acceptedTime and acceptedCallId.

Reply:
So will be:
```
export interface TransferFeatureState {
  transferRequests: TransferRequest[]
}

export interface TransferRequest {
  acceptedTimestamp: Date
  acceptedCallId?: string
}
```
?

Reply:
Agreed. Thanks for the input here. I will merge this in for now. There is still time for some changes here if needed.